File: pub.py
#!usr/bin/env python
# -*- coding: utf-8 -*-

import paho.mqtt.client as mqtt
from time import sleep

#def on_connect(client, userdata, flag, rc):
#  print("Connected with result code " + str(rc))


def on_disconnect(client, userdata, flag, rc):
  if rc != 0:
     print("Unexpected disconnection.")

def on_publish(client, userdata, mid):
  print("publish: {0}".format(mid))


def main():
  client = mqtt.Client()
 # client.on_connect = on_connect
  client.on_disconnect = on_disconnect
  client.on_publish = on_publish

  client.connect("", 1883, 60)

  client.loop_start()


  while True:
      txt = input("メッセージを入力してください>>>")
      client.publish("topic/01", txt)
      sleep(3)

if __name__ == '__main__':
  main()

File: pub.js
var mqtt = require('mqtt');
var client = mqtt.connect('mqtt://118.27.0.235:1883');

client.on('connect', function(){
  console.log('publisher.connected.');
});

setInterval(function(){
  //var message = Date.now().toString()/
  var message = "Hello!"
  client.publish("topic/01", message);
  console.log('publisher.publish:', message);
}, 1000);